content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -90,18 +90,6 @@ Follow our instructions on how to
 
```

Comment:
Similarly here, you have to update the text in the step if you remove the output:
 1.  To ensure the following Kubernetes services are deployed: `istio-pilot`,
     `istio-ingressgateway`, `istio-policy`, `istio-telemetry`, `prometheus`,
     `istio-galley`, and, optionally, `istio-sidecar-injector`, verify they all have an appropriate `CLUSTER-IP`:

Reply:
Fixed :metal: